src/core/MOM_open_boundary.F90

```diff
@@ -828,6 +987,66 @@ subroutine initialize_segment_data(G, OBC, PF)
 
 end subroutine initialize_segment_data
 
+subroutine initialize_obc_tides(OBC, tide_ref_date, nodal_ref_date, tide_constituent_str)
+  type(ocean_OBC_type), pointer       :: OBC !< Open boundary control structure
+  integer, dimension(3), intent(in) :: tide_ref_date      !< Reference date (t = 0) for tidal forcing.
```

Comment:
All of these date arguments should follow the convention throughout the rest of MOM6 and FMS models more generally and be cast as time_types.